### DISCUSSION TITLE
Welcome to snail-ide.github.io Discussions!

### DISCUSSION THREAD

**Discussion by nmsderp** [Snail-IDE/snail-ide.github.io#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Talk about your forks.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ThatBobo** [comment]
Hello, where is the VM files of this repo

**Reply by redspacecat** [reply]
It's in the snail-ide-vm repository

**Comment by ThatBobo** [comment]
Which file
